src/js/media-mute-button.js

```diff
@@ -45,6 +46,12 @@ slotTemplate.innerHTML = `
   <slot name="high">${highIcon}</slot>
 `;
 
+const updateAriaLabel = (el) => {
+  const muted = el.getAttribute(MediaUIAttributes.MEDIA_VOLUME_LEVEL) === 'off';
+  const label = muted ? verbs.UNMUTE() : verbs.MUTE();
+  el.nativeEl.setAttribute('aria-label', label);
+};
```

Comment:
In general are we moving to a more functional coding-style?

Instead of an instance method on the class: `this.updateAriaLabel()`

fwiw I like the more functional style, I think it can be easier to reason about -- but just want to be explicit about the coding style we're going for in this project.

Reply:
Short version: yeah, I'd say we should be using more functions/abstractions separable from the element classes. I would definitely prefer leaning more on using functions if/when it has a generic purpose that can be independently tested. Particularly with "pure" functions, but also with best practices on impure functions, this also helps us avoid unnecessary coupling/assumptions.